File: server/demos/php/celulares/app/Models/Cliente.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Cliente extends Model
{
    use HasFactory;

    protected $fillable = [
        'nombre', 'apellido', 'email', 'telefono', 'celular', 'dni',
        'direccion', 'ciudad', 'fecha_nacimiento', 'tipo', 'empresa',
        'ruc', 'notas', 'activo',
    ];

    protected $casts = [
        'fecha_nacimiento' => 'date',
        'activo' => 'boolean',
    ];

    public function getNombreCompletoAttribute(): string
    {
        return $this->nombre . ' ' . $this->apellido;
    }

    public function ventas()
    {
        return $this->hasMany(Venta::class);
    }

    public function reparaciones()
    {
        return $this->hasMany(Reparacion::class);
    }

    public function totalCompras(): float
    {
        return $this->ventas()->where('estado', 'completada')->sum('total');
    }
}

File: server/demos/dotnet/minimarket/MiniMarket.Web/Models/Cliente.cs
using System.ComponentModel.DataAnnotations;

namespace MiniMarket.Web.Models
{
    public class Cliente
    {
        [Key]
        public int Id { get; set; }

        [Required]
        [Display(Name = "DNI / RUC")]
        public string Documento { get; set; }

        [Required]
        [Display(Name = "Nombre Completo")]
        public string Nombre { get; set; }

        public string Telefono { get; set; }  // Se quitó el ?
        public string Correo { get; set; }    // Se quitó el ?
        public string Direccion { get; set; } // Se quitó el ?

        public bool Estado { get; set; } = true;
    }
}